docs/api.md

```diff
@@ -56,6 +56,8 @@ It does not modify the component class passed to it; instead, it *returns* a new
 
   If your `mapStateToProps` function is declared as taking two parameters, it will be called with the store state as the first parameter and the props passed to the connected component as the second parameter, and will also be re-invoked whenever the connected component receives new props as determined by shallow equality comparisons.  (The second parameter is normally referred to as `ownProps` by convention.)
 
+  If an object is passed, each function inside it is assumed to be a Redux selector or a Factory function.
```

Comment:
don't forget on line 55 to document the argument as `[`mapStateToProps(state, [ownProps]): stateProps`] \(*Object* or *Function*)`

Reply:
Done! Thanks!